src/pspm_cfg/pspm_cfg_pp_heart_data.m

```diff
@@ -228,9 +228,11 @@
                           'selected method.']};
 
 ppg2hb_chan         = pspm_cfg_channel_selector('peripheral pulse oxymetry');
+ppg2hb_chan.help    = {['Number of peripheral pulse oximetry channel ', ...
+     '(default: last peripheral puls oximetry channel)']};
```

Comment:
the help text is already defined in `pspm_cfg_channel_selector`. Please remove here.

Reply:
Thank you for pointing out this. I have corrected this mistake now.